website/docs/r/managed_disk.html.markdown

```diff
@@ -111,6 +111,8 @@ The following arguments are supported:
 
 * `encryption_settings` - (Optional) A `encryption_settings` block as defined below.
 
+* `hyper_v_generation_version` - (Optional) The generation of HyperV of the Disk. Applicable to OS disks only. Possible values are `V1` and `V2`. Changing this forces a new resource to be created.
```

Comment:
```suggestion
* `hyper_v_generation` - (Optional) The HyperV Generation of the Disk, which can only be set for OS Disks. Possible values are `V1` and `V2`. Changing this forces a new resource to be created.
```

Reply:
Doc updated

Reply:
I added a similar clarification like in `os_type`. This property is similar to `os_type`, which can be specified by user when an image reference is not provided, but will be assigned a value by server from the image if it is specified.
I'm thinking if we need to add more comments to this property as well as `os_type` explaining when to use `ignore_changes` for these two properties. What do you think @tombuildsstuff